### PR TITLE
zh-CN: New CR2 translations with line numbers removed

### PR DESCRIPTION
**Pull request by henrif75**
Publishes the new translations for CR2 (originally from zh-CN-cr2) #1463 which were previously reviewed.
It also removes line numbers (#1753)
#324

You can skim this PR with the [GitHub CLI](https://cli.github.com/):

```
gh pr diff 1764 | bat -l patch
```

### CODE REVIEW

**Comment by AgainstEntropy** [comment]
> Publishes the new translations for CR2 (originally from zh-CN-cr2) #1463 which were previously reviewed. It also removes line numbers (#1753) #324
> 
> You can skim this PR with the [GitHub CLI](https://cli.github.com/):
> 
> ```
> gh pr diff 1764 | bat -l patch
> ```

Hi @henrif75, thanks for your effort!

Since these translations have been reviewed before, I can directly approve the merge (after a rough re-review), right?

Then we can continue with the rest of the zh-CN translation on top of the merge result.

**Review by AgainstEntropy — APPROVED** [review]
Merge previously reviewed tranlations (#1463 )

**Comment by henrif75** [comment]
Thanks @AgainstEntropy !

**Pull request merged**
